
From: <jacekpoplawski@wp.pl>

Name of device has been changed in 2.6.9-rc1 to "SkyStar2", but module stv0299
still compares name with "Technisat SkyStar2 driver", strings are different,
and result is that stv0299 detects invalid tuner type.

Cc: Johannes Stezenbach <js@linuxtv.org>

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/media/dvb/frontends/stv0299.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/media/dvb/frontends/stv0299.c~stv0299-device-naming-fix drivers/media/dvb/frontends/stv0299.c
--- 25/drivers/media/dvb/frontends/stv0299.c~stv0299-device-naming-fix	Mon Aug 30 14:44:16 2004
+++ 25-akpm/drivers/media/dvb/frontends/stv0299.c	Mon Aug 30 14:44:16 2004
@@ -1271,7 +1271,7 @@ static long probe_tuner (struct dvb_i2c_
 
 	printk ("%s: try to attach to %s\n", __FUNCTION__, adapter->name);
 
-	if ( strcmp(adapter->name, "Technisat SkyStar2 driver") == 0 )
+	if ( strcmp(adapter->name, "SkyStar2") == 0 )
 	{
 	    printk ("%s: setup for tuner Samsung TBMU24112IMB\n", __FILE__);
 
_
